Makefile

```diff
@@ -40,6 +40,10 @@ acceptance: format vet
 	@echo "> Running acceptance tests..."
 	$(GOCMD) test -mod=vendor -v -count=1 -parallel=1 -timeout=0 -tags=acceptance ./acceptance
 
+acceptance-all: format vet
+	@echo "> Running acceptance tests..."
+	ACCEPTANCE_SUITE_CONFIG=$$(cat ./acceptance/testconfig/all.json) $(GOCMD) test -mod=vendor -v -count=1 -parallel=1 -timeout=0 -tags=acceptance ./acceptance
```

Comment:
@ameyer-pivotal mentioned that there might have been some discussions about this being a path to a file instead of json structure. Is there a preference?

Reply:
Lets not hang this PR up on this question or the test config question. But I would like to improve the experience of running the compatibility tests in the future.